sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/ManagementOperationsLiveTests.cs

```diff
@@ -0,0 +1,250 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.DocumentAnalysis.Tests
+{
+    /// <summary>
+    /// The suite of tests for the <see cref="DocumentModelAdministrationClient"/> class.
+    /// </summary>
+    /// <remarks>
+    /// These tests have a dependency on live Azure services and may incur costs for the associated
+    /// Azure subscription.
+    /// </remarks>
+    [IgnoreServiceError(400, "InvalidRequest", Message = "Content is not accessible: Invalid data URL", Reason = "https://github.com/Azure/azure-sdk-for-net/issues/28923")]
+    public class ManagementOperationsLiveTests : DocumentAnalysisLiveTestBase
```

Comment:
This name has me confused. Why are we testing management operations?

Reply:
You convinced me the name is not good. They're not "management" operations as in Azure management/data-plane. It's just a category we gave to a specific set of APIs (get/list model operations, and get resource info).

This is a name we use internally in the SDK only to organize things. It's not used by the service nor it's written in any public documentation, so we can rename it to "administration operations" to avoid confusion.